### PR TITLE
Rephrase the last challenge. Add question on how to undo a git-init.

### PR DESCRIPTION
**Pull request by iglpdc**
Addresses the enhancement proposed in #33.

It makes the last challenge (why is bad to create a repo inside a repo) to follow a bit more the lesson theme. More importantly, it adds a question on what to do when you create a repo in the wrong place. This is something that happens quite commonly during this workshops, as learners, who are not yet proficient with the command line, put under version control the wrong folder, e.g. their whole home directory.

### CODE REVIEW

**Comment by gvwilson** [comment]
How *would* you undo the second `git init`?  `rm -rf .git`?

**Comment by iglpdc** [comment]
Yes, `rm -rf .git` is what I was thinking. 

I find often learners that had put all their Desktop/home dir under version control and they only realize when they do a `git status` further in the lesson. As `rm -rf` is risky, specially if you do it around one of these places, I usually go to their laptop to double check they aren't removing something they don't want. Doing this at the beginning of the class, in a more controlled way could save some time later in the lesson and also reinforce the idea of what `git init` actually doing (just creating `.git`, from a simplistic point of view) and that all Git magic is self-contained in this directory.

**Comment by gvwilson** [comment]
How about `mv .git /tmp/this-was-the-git-folder` and then `cd /tmp; rm 
-rf this-was-the-git-folder` with an explanation of why?

**Comment by iglpdc** [comment]
Ok, I've added a short paragraph about this to `instructors.md`

**Comment by iglpdc** [comment]
Could someone else take a look at this?

**Comment by gvwilson** [comment]
@daisieh can you please look at this one?

**Comment by daisieh** [comment]
Why not have the challenge itself be "accidentally create a git repo out of your home directory?" Then the warning you're making becomes more explicit.

I'd also recommend, especially for newbies to the command line, that the way to recover is what @gvwilson said: `mv .git tempgit` (it doesn't even have to be out of the directory, just changed to something visible) and then they can look at what the git directory looks like a little before deleting it, just like any other directory.

**Comment by iglpdc** [comment]
Oh, sorry, guys. I did the changes @gvwilson mentioned, but committed them to another branch, that's why I was asking you to review it again... I'll fix it and then you can take a look.

**Comment by iglpdc** [comment]
@daisieh I don't like the idea of making people put their home directory under git and then asking them to use `rm -rf` there: eventually, someone is going to make a mistake and accidentally delete their home dir... 

Also we want to say that making a repo out of your home or your Desktop is not wrong (probably not very useful either), but making repos inside repos is.

**Comment by daisieh** [comment]
Well, not that you should do it, but it could be a thought exercise: like, what would happen if you did it?

File: instructors.md [hunk 1]
```diff
@@ -3,28 +3,171 @@ layout: page
...
+    arise naturally once we start talking about
+    using public services like GitHub to store files;
+    and
+2.  the discussion gives learners a chance to catch their breath
+    after what is often a frustrating couple of hours.
 
-## [A Better Kind of Backup](01-backup.html)
+## Teaching Notes
+
+<<<<<<< HEAD
```
Comment (daisieh) [inline instructors.md]: a merge conflict note in the PR??

**Comment by daisieh** [comment]
@iglpdc, are there supposed to be this many files committed in this PR?

**Comment by iglpdc** [comment]
This is messed up. I'm closing this without merging and adding the changes again.

**Pull request closed without merging**